plugins/woocommerce-admin/client/wp-admin-scripts/product-tracking/shared.ts

```diff
@@ -418,10 +418,69 @@ export const initProductScreenTracks = () => {
 				'.woocommerce_attribute'
 			).length;
 			if ( newAttributesCount > attributesCount ) {
```

Comment:
Thinking more about this, I realized it could made sense to add Name and Slug here and then used those also when adding relevant attribute terms ([here](https://github.com/woocommerce/woocommerce/blob/9d7e503519f96ca6229445f3e1bf1e59a3d956fa/plugins/woocommerce-admin/client/wp-admin-scripts/product-tracking/shared.ts#L386) and [here](https://github.com/woocommerce/woocommerce/blob/9d7e503519f96ca6229445f3e1bf1e59a3d956fa/plugins/woocommerce-admin/client/wp-admin-scripts/add-term-tracking/index.js#L37)).